libs/blocks/bulk-publish/bulk-publish.js

```diff
@@ -87,6 +89,24 @@ function bulkPreviewStatus(row) {
   `;
 }
 
+function bulkDeleteStatus(row) {
+  const status = row.status.delete === 403
+    ? 'Failed to Delete (Probably because the resource still exists)'
```

Comment:
I'm not sure what causes 403s, but do you think this would make more sense?
```suggestion
    ? 'Failed to Delete (Ensure the resource is deleted in Sharepoint)'
```

Reply:
I think the new message is clearer. 
If the user is logged in and has permissions(as @gwatanab tried) the page can be deleted even if the resource already exists. Should that also be expressed in the message?